### DISCUSSION TITLE
[2021.08.25]  베스트앨범

### DISCUSSION THREAD

**Discussion by leechang4** [pyunji/Hyundai-ITnE-Team_QuadCore#39] [Algorithm]
[베스트앨범](https://programmers.co.kr/learn/courses/30/lessons/42579)

**Comment by pyunji** [comment]
[베스트앨범 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/pyunji/pyunji/PR/PR_42579.java)

**Reply by hyk339** [reply]
너무 어렵습니당..쉬는시간에 계속 풀어볼게요!

**Reply by pyunji** [reply]
화이팅 입니다 !

**Reply by chaeyeon4u** [reply]
현지님 풀이 보고 다시 풀어도 어렵네요ㅠㅠ

**Reply by chaeyeon4u** [reply]
다시 풀어도 어려웡

**Comment by hyk339** [comment]
[베스트앨범 solved](https://github.com/hyk339/Hyundai-ITnE-Team_QuadCore/blob/master/khy/programmers/PR_42579.java)

**Reply by hyk339** [reply]
막 풀어보다보니..정리되지 못하게 풀었습니다. 계속 다시 보면서 수정해나가겠습니다^^

**Comment by chaeyeon4u** [comment]
[베스트앨범 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/ycy/homework/Java/PR_42579.java)

**Comment by leechang4** [comment]
[베스트앨범 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/lch/Programmers/PR_42579.java)
